app/models/spree_i18n/translatable.rb

```diff
@@ -1,8 +1,43 @@
 module SpreeI18n
   module Translatable
     extend ActiveSupport::Concern
-    included do
+
+    included do |klass|
       accepts_nested_attributes_for :translations
+      klass.class_eval do
+        class << self
+          alias_method_chain :ransack, :translations
```

Comment:
I needed `alias_method_chain` for my first implementation, but I can now move back to using super `which` is indeed cleaner.